### PR TITLE
Add disable state to variable tag component

### PR DESCRIPTION
**Pull request by thomtrp**
- add hover on variables
- add disable state with readonly props

Disabled
Capture d’écran 2024-11-19 à 16 12 10

Normal
Capture d’écran 2024-11-19 à 16 12 30

### CODE REVIEW

File: packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx [hunk 1]
```diff
@@ -201,11 +215,18 @@ export const VariableTagInput = ({
     <StyledContainer>
       {label && <StyledLabel>{label}</StyledLabel>}
       <StyledInputContainer multiline={!!multiline}>
-        <StyledEditor multiline={!!multiline}>
+        <StyledEditor multiline={!!multiline} readonly={!!readonly}>
```
Comment (martmull) [inline packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx:218]: set readonly optional so you don't need the `!!`

File: packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx [hunk 2]
```diff
@@ -37,26 +39,39 @@ const StyledInputContainer = styled.div<{ multiline: boolean }>`
...
+      : `
+        background-color: ${theme.background.transparent.lighter};
+        border-top-right-radius: ${theme.border.radius.sm};
+        border-bottom-right-radius: ${theme.border.radius.sm};
+        border: 1px solid ${theme.border.color.medium};
+      `}
 `;
 
-const StyledEditor = styled.div<{ multiline: boolean }>`
+const StyledEditor = styled.div<{ multiline: boolean; readonly: boolean }>`
```
Comment (martmull) [inline packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx:74]:
```suggestion
const StyledEditor = styled.div<{ multiline: boolean; readonly?: boolean }>`
```

**Review by martmull — APPROVED** [review]
Small comment but LGTM

File: packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx (hunk 1 above)
Comment (thomtrp) [reply packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx:218]: you are right

**Pull request merged**